mdata/notifierNsq/cfg.go

```diff
@@ -4,31 +4,34 @@
 // and refer to the config variables (clNSQ.Enabled etc), so we currently have this hybrid approach
 // with config handling in this package, but the rest of clNSQ is in the mdata package
 
-package clnsq
+package notifierNsq
 
 import (
 	"flag"
 	"log"
 	"strings"
 
 	"github.com/nsqio/go-nsq"
+	"github.com/raintank/met"
 	"github.com/raintank/misc/app"
 	"github.com/rakyll/globalconf"
 )
 
 var (
-	Enabled          bool
-	NsqdTCPAddrs     string
-	LookupdHTTPAddrs string
-	NsqdAdds         []string
-	LookupdAdds      []string
-	Topic            string
-	Channel          string
-	MaxInFlight      int
-	ProducerOpts     string
-	ConsumerOpts     string
-	PCfg             *nsq.Config
-	CCfg             *nsq.Config
+	Enabled           bool
+	NsqdTCPAddrs      string
+	LookupdHTTPAddrs  string
+	NsqdAdds          []string
+	LookupdAdds       []string
+	Topic             string
+	Channel           string
+	MaxInFlight       int
+	ProducerOpts      string
+	ConsumerOpts      string
+	PCfg              *nsq.Config
+	CCfg              *nsq.Config
```

Comment:
with the exception of Enabled, all these variables don't need to exported anymore because we only use them in the package itself. in fact we probably also don't need the two separate files anymore. everything that's in cfg.go can be moved into notifierNsq.go (although you can also keep it separate if you want, i don't care much)